src/controllers/words.js

```diff
@@ -105,8 +117,26 @@ export const getWords = async (req, res, next) => {
   } catch (err) {
     return next(err);
   }
+}
+/* Gets words from MongoDB */
+export const getWords = async (req, res, next) => {
+  try{
+      getWordsBaseFunction(req, res, next)
+  }catch(err) {
+    return next(err);
+  }
 };
 
+/* Gets words from MongoDB by the searched keywords and specified wordClass */
+export const getWordsFilteredByWordClass =  async (req, res, next) =>{
+  try{
+      let wordClass = parseWordClass(req.params.wordClass);
```

Comment:
Use `const` instead of `let` since you are not reassigning a value to `wordClass`